common/persistence/src/test/java/app/coronawarn/server/common/persistence/domain/DiagnosisKeyBuilderTest.java

```diff
@@ -20,17 +20,19 @@
 
 package app.coronawarn.server.common.persistence.domain;
 
+import static app.coronawarn.server.common.persistence.domain.validation.ValidSubmissionTimestampValidator.SECONDS_PER_HOUR;
 import static org.assertj.core.api.Assertions.assertThat;
 import static org.assertj.core.api.Assertions.assertThatCode;
 import static org.assertj.core.api.Assertions.catchThrowable;
-
```

Comment:
At least in my intelliJ (due to the entries in ``.editorconfig``) this line would get reverted once the formatter is applied. My apologies for the somewhat intransparent situation here.

Reply:
yes I saw https://github.com/corona-warn-app/cwa-server/issues/241 but didn't get the .editorconfig to work with the eclipse formatter . I fixed it manually now